client/rpmtrans.c

```diff
@@ -65,6 +66,18 @@ TDNFRpmExecTransaction(
     }
 
     ts.nTransFlags = rpmtsSetFlags (ts.pTS, RPMTRANS_FLAG_NONE);
+    for (pSetOpt = pTdnf->pArgs->pSetOpt; pSetOpt; pSetOpt = pSetOpt->pNext)
+      {
+        if (pSetOpt->nType != CMDOPT_KEYVALUE)
+          {
+            continue;
+          }
+
+        if (!strcasecmp(pSetOpt->pszOptName, "tsflags") && !strcasecmp(pSetOpt->pszOptValue, "noscripts"))
```

Comment:
Please break this line into two.

Reply:
This look up in O(n).  Why not use a hashtable to store all the options doing in O(1) ? Now I tweaked a bit so that this condition should not occur. More meant for optimization.

Reply:
My request is just for formatting - the line is pretty long, so it should be broken to make it easier to read.

O(n) is not a problem here - this code is not time critical, and there aren't that many options to parse in a typical command line.

Reply:
It's not abt one option or tons. We need to do that